webui/src/lib/components/repository/refDropdown.jsx

```diff
@@ -36,21 +50,31 @@ const BranchSelector = ({ repo, selected, branches, listBranches, selectRef, wit
             </Form>
         </div>
     );
+    const refTypeNav = withTags && <Nav variant="tabs" onSelect={setRefType} activeKey={refType}>
+        <Nav.Item>
+            <Nav.Link eventKey={"branch"}>Branches</Nav.Link>
+        </Nav.Item>
+        <Nav.Item>
+            <Nav.Link eventKey={"tag"}>Tags</Nav.Link>
+        </Nav.Item>
+    </Nav>
```

Comment:
I'm not sure how to fix this but when the RefSelector shows a selected tag, and I click it, the refSelector state does not show "Tags" as the active key. This is what I would expect it to do.